content/tutorial/tutorial.md

```diff
@@ -12,125 +12,130 @@ redirect_from:
   - "docs/tutorial-zh-CN.html"
 ---
 
-This tutorial doesn't assume any existing React knowledge.
+Deze tutorial vereist geen voorbestaande React kennis.
 
-## Before We Start the Tutorial {#before-we-start-the-tutorial}
+## Voordat we beginnen {#voordat-we-beginnen}
 
-We will build a small game during this tutorial. **You might be tempted to skip it because you're not building games -- but give it a chance.** The techniques you'll learn in the tutorial are fundamental to building any React apps, and mastering it will give you a deep understanding of React.
+
+We zullen een klein spelletje bouwen in deze tutorial. **Misschien ben je geneigd om dit over te slaan aangezien je geen games maakt -- maar probeer het.** De technieken die je zal leren in deze tutorial zijn fundamenteel om React applicaties te bouwen, eens je deze onder de knie krijgt, heb je een diep inzicht in React. 
 
 >Tip
 >
->This tutorial is designed for people who prefer to **learn by doing**. If you prefer learning concepts from the ground up, check out our [step-by-step guide](/docs/hello-world.html). You might find this tutorial and the guide complementary to each other.
+>Deze tutorial is bedoelt voor degene die het liefst **al doende leren**. Indien je liever concepten vanuit de basis leert, neem dan een kijkje naar de [stapsgewijze documentatie](/docs/hello-world.html).
```

Comment:
`degene` is the singular form so then it should be `al doende leert`. I do think keeping it plural here is fine so then we'll move to `degenen` (https://onzetaal.nl/taaladvies/degene-diegene). Also moving `tutorial` to `inleiding`

```suggestion
>Deze inleiding is bedoelt voor degenen die het liefst **al doende leren**. Indien je liever concepten vanuit de basis leert, neem dan een kijkje naar de [stapsgewijze documentatie](/docs/hello-world.html).
```